building.sk

```diff
@@ -743,7 +744,6 @@ caves and cliffs update part 1:
 	tuff = minecraft:tuff
 	dripstone block¦s = minecraft:dripstone_block
 	smooth basalt = minecraft:smooth_basalt
-	rooted dirt [block¦s] = minecraft:rooted_dirt
```

Comment:
I noticed you moved this to the other file, but alias placement is generally based on where the item is located within the creative inventory. If rooted dirt is in the building category, it should remain in this file :)

Reply:
Naw, it's in the natural section now. I doubt it was ever in the building category. It's moved because that's where the other dirt is and due to the `any dirt` aliases now, it's needed for order of operations.